src/exampleTypes/Date.js

```diff
@@ -7,118 +7,127 @@ import F from 'futil-js'
 import _ from 'lodash/fp'
 
 let allRollingOpts = [
-  { type: 'all', label: 'All Dates', value: { from: '', to: '' } },
-  { type: 'past', label: 'Last 3 Days', value: { from: 'now-3d', to: 'now' } },
-  { type: 'past', label: 'Last 7 Days', value: { from: 'now-7d', to: 'now' } },
+  { type: 'all', label: 'All Dates', range: { from: '', to: '' } },
```

Comment:
This change (`value` -> `range`) isn't necessary, but with the `option` and `select` already having `value` props, I thought this would be less confusing, since otherwise it'd appear as if we were taking this field unmodified to be the option's value.